### PR TITLE
Allow `actionpack` 6

### PR DESCRIPTION
**Pull request by jacobbednarz**
Updates the gemspec to allow using `actionpack` with Rails 6 for
testing.

### CODE REVIEW

File: actionpack-action_caching.gemspec [hunk 1]
```diff
@@ -14,8 +14,8 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.license       = "MIT"
 
-  gem.add_dependency "actionpack", ">= 4.0.0", "< 6"
+  gem.add_dependency "actionpack", ">= 4.0.0", "< 7"
```
Comment (rafaelfranca) [inline actionpack-action_caching.gemspec]: Can you remove the upper limit?

```suggestion
  gem.add_dependency "actionpack", ">= 4.0.0"
```

File: actionpack-action_caching.gemspec [hunk 2]
```diff
@@ -14,8 +14,8 @@ Gem::Specification.new do |gem|
...
 
   gem.add_development_dependency "mocha"
-  gem.add_development_dependency "activerecord", ">= 4.0.0", "< 6"
+  gem.add_development_dependency "activerecord", ">= 4.0.0", "< 7"
```
Comment (rafaelfranca) [inline actionpack-action_caching.gemspec]:
```suggestion
  gem.add_development_dependency "activerecord", ">= 4.0.0"
```

**Comment by jacobbednarz** [comment]
Sounds great to me!

**Comment by jacobbednarz** [comment]
🙇 thanks a bunch @rafaelfranca !

**Pull request merged**